packages/rockets-next-web/src/app/showroom/users/page.tsx

```diff
@@ -0,0 +1,319 @@
+"use client";
+
+import type { RowProps } from "@concepta/react-material-ui/dist/components/Table/types";
+import type { IChangeEvent } from "@rjsf/core";
+
+import { type FC, useState, useCallback, useMemo } from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import IconButton from "@mui/material/IconButton";
+import Drawer from "@mui/material/Drawer";
```

Comment:
We have a Drawer component on Rockets

Reply:
Christiano is making an implementation on Rockets, please align with him

Reply:
@andreneto97 I think @brunomous just wants to showcase the Form itself. 
The Drawer we've got in Rockets doesn't let @brunomous achieve the desired opening/closing effect. We'd need to tweak a lot of things in the Drawer component from Rockets. I was working on ContainerWithDrawer.